packages/api/src/hooks/useCFDCompareAccounts.tsx

```diff
@@ -69,7 +70,11 @@ const ctraderAccount = {
 /** A custom hook that gets compare accounts values. */
 const useCFDCompareAccounts = () => {
     const { data: activeWallet } = useActiveWalletAccount();
-    const { is_virtual: isDemo } = activeWallet || {};
+    const { data: activeTradingAccount } = useActiveTradingAccount();
+    const { is_virtual: isDemoWallet } = activeWallet ?? {};
+    const { is_virtual: isDemoTrading } = activeTradingAccount ?? {};
```

Comment:
It was only catering for wallets, thats why we were seeing BVI, Vanuatu jurisdictions in demo compare accounts